src/main/java/com/minelittlepony/model/components/BatWings.java

```diff
@@ -13,10 +13,8 @@ public BatWings(T model, float yOffset, float stretch) {
 
     @Override
     public void init(float yOffset, float stretch) {
-        int x = 57;
-
-        leftWing = new ModelBatWing<T>(pegasus, false, false, yOffset, stretch, x, 16);
-        rightWing = new ModelBatWing<T>(pegasus, true, false, yOffset, stretch, x - 1, 16);
+        leftWing = new ModelBatWing<>(pegasus, false, false, yOffset, stretch, 16);
+        rightWing = new ModelBatWing<>(pegasus, true, false, yOffset, stretch, 16);
```

Comment:
I'd like to keep the x parameter on wings, in case we need to put them somewhere else in the future.

Reply:
But why? That parameter only affected extended wings anyway. If you would want to move the wings on pegasi - you would edit `ModelWing`. If you do it on something else - you would either call .tex() in `addFeathers()` (like you do in all existing wing subclasses) or just override the constructor.

Reply:
It's just a thin to consider  You don't gave to, I'd just rather they stay to preserve the idea of it being an object with a position.

Reply:
Well, I could move the `56` from `ModelWing`'s constructor to every call of said constructor, if that makes you happier :)

Reply:
It's no problem, you can leave it then.